tests/testthat/setup-bugRzilla.R

```diff
@@ -12,10 +13,68 @@ if (!nzchar(Sys.getenv("R_BUGZILLA"))) {
 
 use_key()
 
+test_that("create_bugzilla_key works", {
+  vcr::use_cassette("create_bugzilla_key", {
+    host <- "https://bugs.r-project.org/bugzilla/"
+    cbk <- create_bugzilla_key(host)
+  })
+  expect_equal(a <- paste0(host, "userprefs.cgi?tab=apikey"), url)
+  cli::test_that_cli("cli_alert_warning works", {
+    expect_snapshot({
+      cli::cli_alert_warning("This package hasn't been tested with this Bugzilla version.")
+    })
+  })
+})
+
+
+test_that("check_authentication works", {
+  vcr::use_cassette("check_authentication", {
+    ca <- check_authentication(host)
```

Comment:
Tests should be self contained (no external data or functions aside from that provided on setup-bugRzilla). Also, the host parameter can be omitted as it is internally handled by `missing_host`.